source/_includes/asides/component_navigation.html

```diff
@@ -21,7 +22,12 @@
   <div class='section'>
     The {{ page.name | default: page.title }} integration was introduced in Home Assistant {{ page.ha_release | default: "unknown" }},
     and it's used by <a title="Open analytics.home-assistant.io" href="https://analytics.home-assistant.io" target="_blank" rel="noopener">
```

Comment:
Let's link directly to integration stats

```suggestion
    and it's used by <a title="Open analytics.home-assistant.io" href="https://analytics.home-assistant.io/#integrations" target="_blank" rel="noopener">
```